src/Orckestra.Composer.MyAccount/Services/MembershipViewService.cs

```diff
@@ -154,7 +155,26 @@ public virtual async Task<LoginViewModel> LoginAsync(LoginParam param)
             var loginUrl = MyAccountUrlProvider.GetLoginUrl(new BaseUrlParameter { CultureInfo = param.CultureInfo, ReturnUrl = param.ReturnUrl });
             var userName = GenerateUserName(param.Username);
 
-            var loginResponse = Membership.LoginUser(userName, param.Password);
+            var loginResponse = false;
+            try
+            {
+                loginResponse = Membership.LoginUser(userName, param.Password);
+            }
+            catch (ComposerException e)
+            {
+                if (e.Errors[0].ErrorCode == "UserLockedDown")
```

Comment:
move the error code value to the constants in Orckestra.Composer.Utils.Constants